notebooks/MAST/m_dwarf_flares_hst/m_dwarf_flares_hst.ipynb

```diff
@@ -0,0 +1,1412 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    J2000 = Time('2000-1-1 00:00:00.0') # Arjun note — I don't think this time is actually correct!
</pre><p>Gah! That's what I would have thought. But the wikipedia page for "Epoch (astronomy)" implies that's true for TT but not UTC. If you specify scale='TT' is it right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>